### PR TITLE
fix: show logs via command palette

### PR DESCRIPTION
**Pull request by gak**

### CODE REVIEW

File: extensions/vscode/package.json [hunk 1]
```diff
@@ -60,12 +60,14 @@
         "category": "FTL"
       },
       {
-        "command": "ftl.showPanel",
-        "title": "Show Panel"
```
Comment (deniseli) [inline extensions/vscode/package.json:64]: Did you mean to remove Show Panel? Just confirming since I wasn't sure :)

**Review by deniseli — APPROVED** [review]
Nice!

File: extensions/vscode/package.json (hunk 1 above)
Comment (gak) [reply extensions/vscode/package.json:64]: Yeah the bug was that it needs to be showLogs instead. I should have been more clear on why I changed it!

**Review by alecthomas — APPROVED** [review]

**Review by wesbillman — APPROVED** [review]
I did this in my hackathon, but I also want to move all these commands/handlers to a `./commands` folder to have this managed in a structured way. Hoping to get a PR with that refactor soon.

File: extensions/vscode/package.json (hunk 1 above)
Comment (deniseli) [reply extensions/vscode/package.json:64]: Ahhh sweet thanks a bunch for the context :)

**Pull request merged**